src/frontend/komponenter/TilordnetSaksbehandler/TilordnetSaksbehandlerVenstremeny.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { Alert, BodyShort, HStack } from '@navikt/ds-react';
+import { AGray50, AWhite } from '@navikt/ds-tokens/dist/tokens';
+
+import { utledVisningsnavn } from './tilordnetSaksbehandlerUtils';
+import { useBehandling } from '../../context/BehandlingContext';
+import { BehandlingType } from '../../typer/behandling/behandlingType';
+import { SaksbehandlerRolle } from '../../typer/behandling/saksbehandlerDto';
+
+const TilordnetSaksbehandlerVenstremeny: React.FC = () => {
+    const { behandling } = useBehandling();
+
+    if (!behandling.tilordnetSaksbehandler) {
+        return;
+    }
+
+    return (
+        <>
+            {behandling.tilordnetSaksbehandler?.rolle !==
+                SaksbehandlerRolle.OPPGAVE_FINNES_IKKE && (
+                <HStack
+                    gap={'2'}
+                    align={'center'}
+                    paddingInline={'space-16 space-0'}
+                    paddingBlock={'space-8 space-0'}
+                    style={{
+                        backgroundColor:
+                            behandling.type === BehandlingType.REVURDERING ? AGray50 : AWhite,
```

Comment:
Er dette nødvendig? Oppsummeringen dukker opp med grå bakgrunn på fane to i en førstegangsbehandling